CNKI.js

```diff
@@ -136,6 +137,11 @@ function getItemsFromSearchResults(doc, url, itemInfo) {
 	}
 
 	if (!links.length) {
+		// 018<lyb018@gmail.com>: 20200909
```

Comment:
No need to leave a signature (this duplicates `git blame`!). Applies for similar comments below.

Reply:
OK, I'll fix it.